math/clingo/Portfile

```diff
@@ -3,10 +3,12 @@
 PortSystem          1.0
 PortGroup           github 1.0
 PortGroup           cmake 1.1
+PortGroup           muniversal 1.0
+# posix_memalign
+PortGroup           legacysupport 1.1
 
```

Comment:
you should specify which OSes needs this, so add  `legacysupport.newest_darwin_requires_legacy xx`, where you replace `xx` with the appropriate value

Reply:
It defaults to 10, doesn’t it?

Reply:
doesn't look like it, from the PG:
```
# Newest Darwin version that requires legacy support.
# Currently OS X 10.12 ( Sierra, Darwin 16) due to utimensat, fsgetpath, setattrlistat
set ls_max_darwin_support 16
options legacysupport.newest_darwin_requires_legacy
default legacysupport.newest_darwin_requires_legacy ${ls_max_darwin_support}
```

I'd suggest to just add it regardless as most/many other ports do so that it's clear without opening up the PG.

Reply:
Okay, will add. Do I get it right, that if we need a symbol that exists in 10.6 but not earlier, then `legacysupport.newest_darwin_requires_legacy 10`?

Reply:
I haven't looked at this in detail, but the way I read it that you specify the newest version that needs it. So if 10.6 has the symbols but 10.5 not, then I'd think that 10.5 would be the "newest version that needs it". So that would mean it should be `9`. Of course you can read the PG code more carefully or check what other ports do; or you can just try and see if with the `9` it uses the legacy_support group on 10.5.

see also https://trac.macports.org/wiki/XcodeVersionInfo#macOSVersionMap